config/jetpack-cloud-development.json

```diff
@@ -52,6 +52,7 @@
 		"jetpack/pro-dashboard-jetpack-boost": true,
 		"jetpack/pro-dashboard-monitor-multiple-email-recipients": true,
 		"jetpack/pro-dashboard-monitor-paid-tier": true,
+		"jetpack/pro-dashboard-wpcom-atomic-hosting": true,
```

Comment:
Nitpicking, can we move this below `jetpack/pro-dashboard-monitor-sms-notification` so it doesn't break the order?

Reply:
Done: https://github.com/Automattic/wp-calypso/pull/81056/commits/c24ea1018d069d5fa8d87f42a540d36c41fa15fe